RecoLocalCalo/EcalRecAlgos/interface/DeclsForKernels.h

```diff
@@ -26,10 +26,10 @@
 #include "RecoLocalCalo/EcalRecAlgos/interface/EcalTimeBiasCorrectionsGPU.h"
 #include "RecoLocalCalo/EcalRecAlgos/interface/EcalTimeCalibConstantsGPU.h"
 
-class EcalPulseShape;
+struct EcalPulseShape;
 class EcalSampleMask;
 class EcalTimeBiasCorrections;
-class EcalPulseCovariance;
+struct EcalPulseCovariance;
```

Comment:
@vkhristenko Does this (and below) conflict with your ECAL developments?

Reply:
if you mean forward decl changing class -> struct - should be fine. i see original one is a struct, therefore definitely should be fine.

Reply:
I was more thinking if your PR/developments are touching the same lines (for whatever reason) and thus leading to git merge conflicts.

Reply:
Ah, forward decls should be fine (line-wise).